chesster.js

```diff
@@ -300,14 +301,13 @@ function sayCaptains(self, convo, callback){
 /* rating */
 
 controller.hears([
-	'rating'
+    players.appendPlayerRegex("rating", true)	
```

Comment:
I cant put a comment on the line so I'll note it here: error on line 320 can be removed.